src/documentation/netex_idfm_to_ntfs.md

```diff
@@ -195,81 +195,86 @@ comment_id | Line/noticeAssignments/NoticeAssignment/NoticeRef | This field is p
 
 
 ## Reading of each folder
-In a **offre_** file, 2 **GeneralFrame** are expected in a **CompositeFrame/frames** node:
+In a **offre_*.xml** file, 2 **GeneralFrame** are expected in a **CompositeFrame/frames** node:
 * one with a `TypeOfFrameRef/@ref` containing the string `NETEX_STRUCTURE`
 * one with a `TypeOfFrameRef/@ref` containing the string `NETEX_HORAIRE`
 In the following chapters, the **CompositeFrame/frames/GeneralFrame/members** is ommitted.
```

Comment:
First of all,
```suggestion
In the following chapters, the **CompositeFrame/frames/GeneralFrame[]/members** is ommitted.
```
But also, this sentence is not entirely true because only 2 lines below, you're using the full path...

Reply:
you're right, I changed my writing and forgot to remove this one ! thanks